webapp/src/index.css

```diff
@@ -32,12 +32,7 @@
 /* Styles for viewports wider than 768 pixels. */
 @media (min-width: 768px) {
   :root {
-    --title-font-size: 3rem;
-    --subtitle-font-size: 2.5rem;
-    --heading-font-size: 2rem;
-    --subheading-font-size: 1.5rem;
-    --paragraph-font-size: 1.25rem;
-    --caption-font-size: 1rem;
+    font-size: 18px;
```

Comment:
Increase base `px` font size for desktop widths

Reply:
As said in a comment bellow, please re-check Figma Library I thinks we are looking to different files! I've already made changes in the "official" library in `ft_transcendence` team -> https://www.figma.com/file/0es84Oh1roPphFOd2GYysn/Library?node-id=157%3A26

I think we should remove "deprecated" team and files in Figma so we don't get confused again, this has been a recurrent issue along several PRs :sweat_smile: 


About the minimal difference. I think it matches well despite being so little. I have been testing it on mobile and desktop. If we place a smaller root font for mobile, text may be too small, if we place it bigger for desktop, too big. That being said, we could discuss about this on next meeting because is a matter of personal taste :yum: I'll add a point